plugin/bitly/client.py

```diff
@@ -47,3 +49,19 @@ def get_link_clicks(self, link_id: str):
                 f"Failed to get bilink clicks summary for link id '{link_id}': {resp.text}"
             )
         return resp.json()["link_clicks"]
+
+    def get_link_countries_clicks(self, link_id: str, unit: str):
+        resp = self._get(f"bitlinks/{link_id}/countries?unit={unit}&units=1")
```

Comment:
Are you not making the units configurable? I suppose that makes the datetime rounding more complicated.

Reply:
Yeah, I thought about it, but then I really did not find any value in this. It does not make sense to run the sync every three days and actually have it working with the incremental tables. To collect historical statistics for every period(unit) It really works only if you run at a higher frequency than your unit and the number of units needs to always be set to `1`. I'd wait until we get feedback or we actually find out we need a bit more flexibility here.

Reply:
Sounds good